docs/configuration/data-validation.md

```diff
@@ -94,17 +94,17 @@ The following colors are supported:
 
 !!! tip "Dynamic Configuration Parameter"
 
-This is a mapping of models to [custom validators](../customization/custom-validation.md) against which an object is evaluated immediately prior to its deletion. If validation fails, the object is not deleted. An example is provided below:
+This is a mapping of models to [custom validators](../customization/custom-validation.md) against which an object is evaluated immediately prior to its deletion. If validation fails, the object is not deleted. The configuration can be carried out in the menu under “System/Configuration History/Custom validators”. 
+In the example, a device can only be deleted when it is set to the “decommissioning” status. Paths such as “dcim.device” can be found in the top right-hand corner when you open the page of a device. For a rack it would therefore be “dcim.rack”.
 
 ```python
-PROTECTION_RULES = {
```

Comment:
There's no reason to remove this.

Reply:
I think the example is not well suited or not well described for someone who deals with the topic.

If I understand it correctly, it mixes 3 complexities together.

Example: The site can only be deleted if the status is: decommissioning. 

**Notation for Configuration.py file:**

PROTECTION_RULES = {
    ‘dcim.site": [
        {
            ‘status": {
                ‘eq": ’decommissioning’
            }
        },
    ]
}

**Notation for web frontend:**

‘dcim.site": [
    {
        ‘status": {
            ‘eq": ’decommissioning’
        }
    }
 ]

**Notation for Configuration.py with further validation of custom plugins:**

PROTECTION_RULES = {
    ‘dcim.site": [
        {
            ‘status": {
                ‘eq": ’decommissioning’
            }
        },
        ‘my_plugin.validators.Validator1’,
    ]
}

The problem with the original example is that it does not work out of the box.
Only after many conversations do you find out what is too much.

**‘my_plugin.validators.Validator1’,:** 
Does not work at all for beginners. It is something for advanced users.
**Result:** An example with this line should be listed separately. As it prevents the part from not working that would always work.

**PROTECTION_RULES = {:** 
Where does it say that this must be omitted in the web frontend? After being told that you can insert this example into the web frontend, it does not work because ‘PROTECTION_RULES = {’ must be omitted. But as a beginner you don't know that.
**Result:**: If not here, then it must be written somewhere else that the outermost bracket in the web frontend is omitted if it is inserted in the respective parameter fields.

Reply:
I have an idea,

in the chapter ‘Dynamic Configuration Parameter’ I would like to point out that the outer bracket e.g. PROTECTION_RULES = { ..} is relevant within the Configuration.py. In the web frontend, however, this should be removed from the examples. As the parameter in the web front end already represents this.

Would that be ok?